src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -78,6 +79,35 @@ public static Object[] compressionAlgorithms() {
 
   @Parameter public String compressionAlgorithm;
 
+  private static final OffsetTokenVerificationFunction offsetTokenVerificationFunction =
```

Comment:
Should we verify the behaviour of the this verification function by test cases that have/don't have a mismatch?